content/open-source.de.md

```diff
@@ -90,12 +90,15 @@ Ein Teil der Entwicklungsarbeit findet momentan über eine private GitLab-Instan
     <a class="button button-primary read-more-button" href="https://github.com/baltpeter/yace">Zum Repository&nbsp;<span class="icon icon-arrow-right"></span></a>
 </article>
 
+<a id="license-notices"></a>
```

Comment:
hugo does provide anchors on it's own already.

Reply:
Yeah but isn't that some ugly thing? I'd much rather have a clear one that I control.